sharpy/aero/utils/airfoilpolars.py

```diff
@@ -52,11 +54,15 @@ def initialise(self, table):
                 iaoacl0 = imin
         self.aoa_cl0_deg = matches[iaoacl0]
 
-        self.cl_interp = interp1d(self.table[:, 0], self.table[:, 1])
-        self.cd_interp = interp1d(self.table[:, 0], self.table[:, 2])
-        self.cm_interp = interp1d(self.table[:, 0], self.table[:, 3])
+        self.aoa_bounds = [np.min(self.table[:,0]),np.max(self.table[:,0])]
+
+        self.cl_interp = interp1d(self.table[:, 0], self.table[:, 1],bounds_error=False,fill_value=(self.table[0,1],self.table[-1,1]))
+        self.cd_interp = interp1d(self.table[:, 0], self.table[:, 2],bounds_error=False,fill_value=(self.table[0,2],self.table[-1,2]))
+        self.cm_interp = interp1d(self.table[:, 0], self.table[:, 3],bounds_error=False,fill_value=(self.table[0,3],self.table[-1,3]))
```

Comment:
Good points Alvaro. The two different fill values given are for when not only the upper but also the lower end of the input polars are reached. I agree with you that it is easier to edit the polar input file directly to avoid user misunderstandings. I have removed the warnings and instead an error is raised by Scipy's interpolation function (see https://github.com/ImperialCollegeLondon/sharpy/pull/217/commits/737846bfe6f5ba2a33e7cc5e4f8d2dc55d0c83b9)